### PR TITLE
A simple excel-creator tool  

### PR DESCRIPTION
**Pull request by imantahriri**
A simple excel-creator tool that receives an object and creates a Excel file with every key as tilte and all properties as records for each title.

### CODE REVIEW

**Comment by asemoller** [comment]
This pr fixes #41 and is required for the export function "simpleExport" in issue [#516 (origo-map)](https://github.com/origo-map/origo/issues/516).

The service can be called at origoserver/excelcreator.

File: handlers/excelcreator.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+function postHandler(req, res) {
+
+  var workbook = new excel.Workbook(); //creating workbook
+  var sheet = workbook.addWorksheet('MySheet'); //creating worksheet
+
+  var objArray = req.body;
+
+  Object.entries(objArray).forEach(entry => {
+    // console.log(entry[0]);
```
Comment (daRasmussen) [inline handlers/excelcreator.js]: remove?

File: handlers/excelcreator.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  var fileName = './public/temp.xlsx';
+
+  // Timeout is set for testing delay spinners in the client, should be removed after test is complete.
+  setTimeout(() => {
+    workbook.xlsx.writeFile(fileName)
+      .then(function () {
+        console.log("file is written");
+        res.setHeader('content-type', 'application/vnd.ms-excel');
+        // res.status(400);
```
Comment (daRasmussen) [inline handlers/excelcreator.js]: remove?

File: handlers/excelcreator.js (hunk 1 above)
Comment (imantahriri) [reply handlers/excelcreator.js]: done.

File: handlers/excelcreator.js (hunk 2 above)
Comment (imantahriri) [reply handlers/excelcreator.js]: done.

**Comment by asemoller** [comment]
Is this ok now? Is the function generic?

**Comment by tonnyandersson** [comment]
@asemoller, @imantahriri see my review comments.

**Comment by tonnyandersson** [comment]
Also, would it be possible to delete the *public/temp.xlsx* file when the export is completed? Or is it a good thing to have around for some reason?

**Comment by asemoller** [comment]
@tonnyandersson, do you mean your comments about geometry in pr [#861](https://github.com/origo-map/origo/pull/861)? Wasn't that solved in that pr?

**Comment by tonnyandersson** [comment]
@asemoller No, my review comments above.

**Comment by asemoller** [comment]
@tonnyandersson Aha, you mean your comment about the temp-file?

**Comment by tonnyandersson** [comment]
@asemoller No, I made two comments regarding specific lines in the code. You can't see them? Perhaps if you go to [Files changed](https://github.com/origo-map/origo-server/pull/49/files)?

The comments were regarding the following lines/blocks in *handlers/excelcreator.js*:

**Line 29:** Line commented out. Should be removed if no longer needed.

**Lines 40-59:** If the `setTimeout()` function was used for testing only, it should be removed. Not the whole block, of course. Just the timer.

And the comment regarding the temp file is still valid. Also, there are some conflicts that must be resolved.

**Comment by asemoller** [comment]
@tonnyandersson Thanks for the clarification. No, I can´t see them, so for a while I thought I had lost it :)!

I'll see when we can get this fixed!

**Pull request merged**